### DISCUSSION TITLE
(vivaldi / vivaldi.portable / vivaldi.install) Shim vivaldi.exe to make it available for use in scripts

### DISCUSSION THREAD

**Discussion by akaleeroy** [pauby/ChocoPackages#205] [Ideas]
Please shim `vivaldi.exe` to `$($env:ChocolateyInstall)\bin` so that scripts can invoke it directly, without having to type the full path.

Because in my install, using default options, it's not shimmed there, whereas other packages have shims in the `bin` folder.

**Comment by pauby** [comment]
To set expectations, this isn't likely to get added for a while as you are the first person every to mention it. PR's are accepted.
